src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import SearchBar from "../SearchBar";
+import { Container } from "react-bootstrap";
+import "./index.scss";
+
+export default function Blogs() {
+  const [blogs, setBlogs] = useState([]);
+  const URL =
+    "https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts";
+  const fetchBlogs = async () => {
+    const data = await fetch(URL);
+    const fetchedBlogs = await data.json();
+    setBlogs(fetchedBlogs);
+    console.log(fetchedBlogs);
+  };
+  useEffect(() => {
+    fetchBlogs();
+  }, []);
+  const NUM_BLOGS_TO_SHOW = 9;
+  const fetchedBlogsArr = blogs.slice(0, NUM_BLOGS_TO_SHOW);
+  console.log(fetchedBlogsArr);
+  return (
+    <div className="blogsMainContainer">
+      <h2 className="ourBlogTitle">Our Blog</h2>
+      <p>
+        Infomration about violence, women and health. Read different topics
+        <br />
+        that contribute positivley to the mental health of women.
+      </p>
+      <SearchBar />
+      <Container>
+        {fetchedBlogsArr.map((blog) => {
+          return (
+            <div className="blogsContainer">
+              <div className="shadowEffect"></div>
+              <h3 className="blogTitles">{blog.title.rendered}</h3>
+              <img
+                className="blogImgs"
+                alt="blog Img"
```

Comment:
Blog image

Reply:
this name caused a problem when running the tests so I had to change it to this name since the tag is already for images

Reply:
in alt? i don't think that `alt="Blog image"` will create any problem

Reply:
![image](https://user-images.githubusercontent.com/62549249/98863502-866f0e80-2479-11eb-84ae-bc2ed4e4ab2c.png)

Reply:
after changing the name in alt, the test passes

Reply:
you can use blog.title as an alt. it won't cause error and will be unique for every blog post.